common/src/main/java/io/netty/util/AbstractConstant.java

```diff
@@ -15,25 +15,23 @@
  */
 package io.netty.util;
 
-import java.util.concurrent.atomic.AtomicLong;
+import java.util.concurrent.atomic.AtomicInteger;
 
 /**
  * Base implementation of {@link Constant}.
  */
 public abstract class AbstractConstant<T extends AbstractConstant<T>> implements Constant<T> {
 
-    private static final AtomicLong uniqueIdGenerator = new AtomicLong();
+    private static final AtomicInteger uniqueIdGenerator = new AtomicInteger();
     private final int id;
     private final String name;
-    private final long uniquifier;
 
     /**
      * Creates a new instance.
      */
-    protected AbstractConstant(int id, String name) {
```

Comment:
@normanmaurer this is protected constructor. All related code changed accordingly. Did I miss something?